project/Doc.scala

```diff
@@ -121,74 +117,90 @@ object UnidocRoot extends AutoPlugin {
     // Override the Scala unidoc target to *not* include the Scala version, since we don't cross-build docs
     target in (ScalaUnidoc, unidoc) := target.value / "unidoc",
     scalacOptions in (ScalaUnidoc, unidoc) ++= Seq("-skip-packages", "com.lightbend.lagom.internal"),
-    javacOptions in doc := Seq(
```

Comment:
Why are these dropped from `javacOptions in doc`?

Reply:
They are _moved_ to `javacOptions in (JavaUnidoc, unidoc)`. I'm not familiar with the internal mechanics of `doc` and `unidoc` but using `sbt unidoc` from JDK11 was ignoring the settings from `javacOptions in doc`.

Setting the argument via `javacOptions in (JavaUnidoc, unidoc)` produces the expected Apidocs HTML in both JDK8 and JDK11.

Reply:
Unless we have reason to drop them from here too, I'd keep them.

Reply:
(Dedupped, of course.)

Reply:
(Dedupped, of course.)